DataStructures.Tests/Heap/MinMaxHeapTests.cs

```diff
@@ -29,15 +36,50 @@ public static void CustomComparerTest()
             mmh.Add(s);
         }
 
-        Assert.AreEqual(comparer, mmh.Comparer);
-        Assert.AreEqual("c", mmh.GetMin());
-        Assert.AreEqual("aaaa", mmh.GetMax());
+        Assert.That(comparer, Is.EqualTo(mmh.Comparer));
+        Assert.That("c", Is.EqualTo(mmh.GetMin()));
+        Assert.That("aaaa", Is.EqualTo(mmh.GetMax()));
+    }
+
+    #region AddTest
+    [TestCaseSource(nameof(CollectionsSourceInt))]
```

Comment:
If going with multiple sources, please rewrite tests like that to reduce duplication:
```
AddTestInt(IEnumerable<int> collection) => AddTest(collection);
AddTestInt(IEnumerable<char> collection) => AddTest(collection);
etc.
AddTest<T>(IEnumerable<T> collection) { test code here }
```